Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -59,31 +59,35 @@ fi
 
 # Checking Ubuntu release to adapt software version to install
 RELEASE_CODENAME=$(lsb_release -c -s)
-PYTHON_V="python"  # starting from ubuntu 20.04, python isn't symlink to default python interpreter
-PIP=pip2
-
-if [ ${RELEASE_CODENAME} == 'xenial' ]; then
-    SITLFML_VERSION="2.3v5"
-    SITLCFML_VERSION="2.3"
-elif [ ${RELEASE_CODENAME} == 'disco' ]; then
+PYTHON_V="python3"  # starting from ubuntu 20.04, python isn't symlink to default python interpreter
```

Comment:
The changes in this file are unrelated to the PR.

Reply:
This was added accidentally.